### PR TITLE
Add ruby testing sample

### PR DESCRIPTION
**Pull request by munisystem**
## WHY

For explain usage simply.
## WHAT

Add sample ruby code using RSpec and GithubAPI.

### CODE REVIEW

**Comment by munisystem** [comment]
@awakia 

> It is not possible to get a wildcard Extended Validation Certificate – instead, all fully qualified domain names must be included in the certificate and inspected by the Certificate Authority 
> https://en.wikipedia.org/wiki/Extended_Validation_Certificate

It is impossible to use trusted certificate with mitm, Because client validate the certificate domain before to send "Client Key Exchange".
And ruby can't configure that all https connection don't checking trusted certificate...
(Like golang net/http "http.DefaultTransport")

So I think to use the webmock-proxy generated root certificate when validation the certificate.
(Client has to download root certificate from webmock-proxy before testing)
ref. https://github.com/wantedly/webmock-proxy/blob/ad22bb7cc6fb96ab974fb855b9c1c54bbb7b2444/example/ruby/spec/spec_helper.rb#L1-L2

What do you think about this? Could you please give some comments?

File: example/ruby/spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+RSpec.configure do |config|
+  config.expect_with :rspec do |expectations|
+    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+  end
+
+  config.mock_with :rspec do |mocks|
+    mocks.verify_partial_doubles = true
+  end
+end
+
```
Comment (awakia) [inline example/ruby/spec/spec_helper.rb:17]: unneeded line

File: example/ruby/sample.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'openssl'
+require './github'
+
```
Comment (awakia) [inline example/ruby/sample.rb:5]: hmm, I think for this short sample it's easier to understand by embedding github.rb into here. How do you think?

Comment (munisystem) [inline example/ruby/sample.rb:5]: You're right. I thought that it is basic implementation app using external APIs. But embedding api client into sample.rb is better.
I'll fix it.

File: example/ruby/github.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+          https.get(uri.request_uri)
+        }
+        JSON.parse(res.body)
+      rescue => e
+        p e.message
+        nil
+      end
+    end
+  end
+end
```
Comment (awakia) [inline example/ruby/github.rb]: please remove this for now

**Comment by awakia** [comment]
LGTM!

**Pull request merged**